fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/InsertIntoTableCommand.java

```diff
@@ -102,6 +102,8 @@ public void setJobId(long jobId) {
 
     @Override
     public void run(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        // insert into setting to disable page cache
+        ctx.getSessionVariable().enablePageCache = false;
```

Comment:
If you modify the code like this. when the insert into select command finished. and user want to execute select command, the page cache value is false too. The performance maybe bad.
I think you could refer the code insert into xxx select /* SET_VAR=xxx*/
to know how to set the session variable at sql level not at session level.